js/notebook/src/plot/plotScope.js

```diff
@@ -262,16 +262,20 @@ define([
       this.plotFocus.focus
     );
 
-    this.calcMapping(false);
+    this.plotRange.calcMapping(false);
     this.update();
   };
 
+  PlotScope.prototype.calcMapping = function(emitFocusUpdate) {
```

Comment:
@mjurowicz-xsolve Is it used anywhere? It looks like it's not needed anymore. But maybe I'm wrong.

Reply:
It's used in `CombinedPlots` yet